block-producer/src/batch_builder/mod.rs

```diff
@@ -80,6 +80,7 @@ where
     // --------------------------------------------------------------------------------------------
 
     /// TODO: add comments
+    #[instrument(target = "miden-block-producer", name = "batch_builder" skip_all)]
```

Comment:
Shouldn't this be:
```
#[instrument(target = "miden-block-producer", name = "block_producer" skip_all)]
```
If we have it as `batch_builder` wouldn't this be the root span's name?

Reply:
But we don't do the same for transaction queue? Why is that different?

Reply:
The only child of this span would be `try_build_block` - right? If so, I think having `block_producer:try_build_block` is clear enough and consistent with how we do it for transaction queue.

Reply:
@bobbinth okay, I agree. :)